koupleless-ext/koupleless-ext-web-gateway/src/test/resources/biz1_forwards.yaml

```
@@ -0,0 +1,8 @@
+- contextPath: test1
+  hosts:
+    - test1
+  paths:
+    - from: /test1
+- contextPath: test1
+  hosts:
+    - test5
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)